input/pagecontent/index.md

```diff
@@ -41,19 +41,21 @@ A system that is the digital interface that allows patients or their representat
 This guide is divided into several pages which are listed at the top of each page in the menu bar.
 
 - [Home](index.html): This page provides the introduction and scope for this guide.
-- [Conformance](conformance.html): This is a placeholder for when content becomes available.
-- [Guidance](guidance.html): This is a placeholder for when content becomes available.
-  - [AU eRequesting data for interoperability](auereqdi.html): This is a placeholder for when content becomes available.
-  - [Relationship with other HL7 AU FHIR IGs](relationship.html): This is a placeholder for when content becomes available.
+- [Conformance](conformance.html): This page describes the set of rules to claim conformance to this guide
+- [Guidance](guidance.html): These pages list the guidance for this guide. 
+  - [AU eRequesting Data for Interoperability](auereqdi.html): This page maps the AU eRequesting resources and elements to AUeReqDI data classes and data elements.
+  - [Relationship with other HL7 AU FHIR IGs](relationship.html): This page provides guidance on the relationship between AU eRequesting and other implementation guides.
```

Comment:
For Relationship with other IGs line:
1. title it "Relationship with other IGs"
2. replace the line with: This page provides guidance on the relationship between AU Core, AUCDI, and other implementation guides.

Reply:
Fixed, thanks.
Also amended title in Guidance page; and menu (sushi-config.yaml file).